--- ./configure.ac	2026-04-18 19:27:00.000000000 +0200
+++ ./configure.ac	2026-04-18 19:29:54.000000000 +0200
@@ -54,6 +54,7 @@
             [if test "x$withval" = "xno"; then ac_cv_header_pcap_pcap_h=no; fi])
 AC_ARG_WITH(pcsclite,  AS_HELP_STRING([--without-pcsclite],[do not use PCSC lite]))
 AC_ARG_WITH(piper,     AS_HELP_STRING([--without-piper],[do not use the Piper TTS library]))
+AC_ARG_WITH(jack2,     AS_HELP_STRING([--without-jack2],[do not use PipeWire sound support]))
 AC_ARG_WITH(pthread,   AS_HELP_STRING([--without-pthread],[do not use the pthread library]))
 AC_ARG_WITH(pulse,     AS_HELP_STRING([--without-pulse],[do not use PulseAudio sound support]))
 AC_ARG_WITH(sane,      AS_HELP_STRING([--without-sane],[do not use SANE (scanner support)]))
@@ -1636,6 +1637,18 @@
       [Rebuilding po files is not supported for out of tree builds.])
 fi
 
+dnl **** Check for Jack2 ****
+if test "x$with_jack2" != "xno";
+then
+    WINE_PACKAGE_FLAGS(JACK2,[libjack],,,,
+        [AC_CHECK_HEADER([jack/jack.h],
+            [AC_CHECK_LIB(jack, jack_client_open,[:],[JACK2_LIBS=""],[$JACK2_LIBS])],
+            [JACK2_LIBS=""])])
+fi
+WINE_NOTICE_WITH(jack2, [test -z "$JACK2_LIBS"],
+        [libjack ${notice_platform}development files not found or too old, WINEASIO won't be supported.],
+        [enable_jack2])
+
 dnl **** Check for PulseAudio ****
 if test "x$with_pulse" != "xno";
 then
@@ -3086,6 +3099,7 @@
 WINE_CONFIG_MAKEFILE(dlls/psapi/tests)
 WINE_CONFIG_MAKEFILE(dlls/pstorec)
 WINE_CONFIG_MAKEFILE(dlls/pstorec/tests)
+WINE_CONFIG_MAKEFILE(dlls/wineasio)
 WINE_CONFIG_MAKEFILE(dlls/pwrshplugin)
 WINE_CONFIG_MAKEFILE(dlls/qasf)
 WINE_CONFIG_MAKEFILE(dlls/qasf/tests)
